src/validation.cpp

```diff
@@ -310,6 +310,9 @@ bool CheckSequenceLocks(const CTransaction &tx, int flags, LockPoints* lp, bool
 }
 
 
+// Returns the script flags which should be checked for a given block
```

Comment:
Why this? if you want to use it between this and its deifinition, why not move the definition here directly?